Models/Comment.cs

```diff
@@ -29,17 +29,26 @@
 using Microsoft.EntityFrameworkCore;
 using Microsoft.Extensions.DependencyInjection;
 using PalaverCore.Data;
+using PalaverCore.Services;
 
 namespace PalaverCore.Models
 {
     public class Comment : TimeStamper
     {
+        public enum TextFormat
+        {
+            HTML = 0,
+            Markdown = 1
+        }
+
         [Key]
         public int Id { get; set; }
         [Required]
-        public string Text { get { return _text; } set { _text = FilterComment(value); } }
```

Comment:
The `Text` property is no longer filtered or modified, the filtering & modification happens only for `DisplayText` on output when being mapped to a view model that needs that property.